frontend/src/pages/ArtistOverview.js

```diff
@@ -84,3 +90,11 @@ const Overview = styled.div`
   }
 
 `
+
+const Loading =styled.section`
```

Comment:
maybe create reusable Loading component? This should save you some lines of code

Reply:
I will have a look into this ...